web/src/app/admin/admin-service-metrics/AdminServiceTable.tsx

```diff
@@ -63,10 +63,12 @@ export default function AdminServiceTable(
       })
     }
 
+    const intKeys = service.integrationKeys
+    const hbMonitors = service.heartbeatMonitors
+
     return {
       hasEPSteps: !!targets.length,
-      hasIntegrations:
-        !!service.integrationKeys.length && !!service.heartbeatMonitors.length,
+      hasIntegrations: !(!intKeys.length && !hbMonitors.length),
```

Comment:
Could we say `service.integrationKeys.length > 0 || service.hearbeatMonitors.length > 0` to be more direct?

Reply:
or

```
const hasIntKeys = !!service.integrationKeys.length
const hasHB = !!service.heartbeatMonitors.length

// ...

hasIntegrations: hasIntKeys || hasHB

// ...
```